modules/data-storage-management.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * storage/optimizing-storage.adoc
+
+[id="data-storage-management_{context}"]
+= Data storage management
+
+The following table summarizes the main directories that {product-title} components write data to.
+
+.Main directories for storing {product-title} data
+[options="header,footer"]
+|===
+|Directory|Notes|Sizing|Expected growth
+
+|*_/var/lib/etcd_*
+|Used for etcd storage when in Multi-Master mode, when etcd is made standalone by an administrator, or when storing the database.
+|Less than 20 GB.
```

Comment:
@hexfusion what do you think about the usage of terms such as Multi-Master mode, and standalone here? 

I think they are vestiges from 3.x versions, should be removed.

Reply:
I don't have strong feels here, obviously, the designation of `control-plane` compute should be inline with current documentation.

Reply:
@retroflexer @hexfusion I've removed references to both Multi-Master mode and standalone:
_Used for etcd storage when storing the database._
^ Is this better?

Reply:
@retroflexer and @hexfusion - Bump, can I get a `lgtm` for this wording, please?